static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -230,6 +232,32 @@ export function StacktraceLink({frame, event, line, disableSetup}: StacktraceLin
     refetch();
   };
 
+  const filePath =
+    frame.filename && frame.lineNo ? frame.filename + ':' + frame.lineNo : '';
+
+  const {onClick: handleCopyPath} = useCopyToClipboard({
+    text: filePath,
+    successMessage: t('File path copied to clipboard'),
+    errorMessage: t('Failed to copy file path'),
+  });
```

Comment:
Most of the additions here aren't relevant to the rest of the contents of the StackTraceLink component, I think it might be better to move it all to a new component similar to `CodecovLink` (e.g. `CopyFrameLink`). 

You wouldnt' have specify the`copyButtonProps` constant either since it'd be one component that gets rendered three times, and it looks like the only props you'd need would be `frame` and `shouldFadeIn` or something like that.

Better separation of concerns that way too :+1: